test/widget_test.dart

```diff
@@ -139,6 +139,28 @@ void main() {
 
       expect(controller.state?.isFront, false);
     });
+
+    testWidgets('manually via controller without animation',
+        (WidgetTester tester) async {
+      final controller = FlipCardController();
+
+      await tester.pumpWidget(
+        Directionality(
+          textDirection: TextDirection.ltr,
+          child: new FlipCard(
+            controller: controller,
+            flipOnTouch: false,
+            front: Text('front'),
+            back: Text('back'),
+          ),
+        ),
+      );
+
+      controller.toggleCardWithoutAnimation();
+      await tester.pumpAndSettle();
```

Comment:
If there's no animation, then a single `.pump()` should be enough, but this is fine too if it isn't enough.